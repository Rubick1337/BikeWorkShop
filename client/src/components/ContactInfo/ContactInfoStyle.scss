.container-contacts-info
{
  display: flex;
  justify-content: center;
  gap: 30px;
  flex-wrap: wrap;
}
.contacts-info
{
  display: flex;
  width: 650px;
  flex-direction: column;
  align-items: flex-start;
  gap: 15px;
}
.container__info
{
  display: flex;
  padding: 30px 0px;
  flex-direction: column;
  align-items: center;
  gap: 49px;
}
.text-contacts h2, .text-contacts a
{
  color: #100;
  font-family: "Noto Sans";
  font-size: 18px;
  font-style: normal;
  font-weight: 400;
  line-height: normal;
  text-decoration: none;
}
@media (max-width: 650px)
{
  .contacts-info
  {
    width: 300px
  }
}