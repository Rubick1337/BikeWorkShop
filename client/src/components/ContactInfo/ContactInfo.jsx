import React from 'react';
import Map from "../Map/Map"
import "./ContactInfoStyle.scss"
const ContactInfo = () => {
    return (
        <div className="container__info">
            <div className="title_section">
                <h3>Контакты</h3>
                <div className="line__title"></div>
            </div>
            <div className="container-contacts-info">
                <div className="contacts-info">
                    <div className="text-contacts">
                        <h2>Центральный офис:<br/>г. Москва, ул,Гоголя 13, 1й этаж</h2>
                    </div>
                    <div className="text-contacts">
                        <h2>Представительство в Киеве:<br/>г. Киев, ул. Є. Коновальця, 103 оф. 410</h2>
                    </div>
                    <div className="text-contacts"><a href="https://github.com/Rubick1337?tab=repositories">Github</a>
                    </div>
                    <div className="telephone">
                        <h2>Телефон:</h2>
                        <div className="text-contacts"><a href="tel:+7987654-54-56">+7 987 654-54-56</a></div>
                        <div className="text-contacts"><a href="tel:+38067504-32-57">+38 067 504-32-57</a></div>
                    </div>
                    <div className="text-contacts"><a
                        href="mailto:poss1337@gmail.com">Почта:&nbsp;poss1337@gmail.com</a></div>
                </div>
                <Map/>
            </div>
        </div>

    );
}

export default ContactInfo;
