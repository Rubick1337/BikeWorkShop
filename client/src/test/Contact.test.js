import React from 'react';
import { render } from '@testing-library/react';
import ContactInfo from '../components/ContactInfo/ContactInfo'; // Убедитесь, что путь правильный

describe('ContactInfo Component', () => {
    it('matches snapshot', () => {
        const { asFragment } = render(<ContactInfo />);
        expect(asFragment()).toMatchSnapshot();
    });
});