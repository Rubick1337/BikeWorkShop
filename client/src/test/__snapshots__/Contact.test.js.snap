// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactInfo Component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container__info"
  >
    <div
      class="title_section"
    >
      <h3>
        Контакты
      </h3>
      <div
        class="line__title"
      />
    </div>
    <div
      class="container-contacts-info"
    >
      <div
        class="contacts-info"
      >
        <div
          class="text-contacts"
        >
          <h2>
            Центральный офис:
            <br />
            г. Москва, ул,Гоголя 13, 1й этаж
          </h2>
        </div>
        <div
          class="text-contacts"
        >
          <h2>
            Представительство в Киеве:
            <br />
            г. Киев, ул. Є. Коновальця, 103 оф. 410
          </h2>
        </div>
        <div
          class="text-contacts"
        >
          <a
            href="https://github.com/Rubick1337?tab=repositories"
          >
            Github
          </a>
        </div>
        <div
          class="telephone"
        >
          <h2>
            Телефон:
          </h2>
          <div
            class="text-contacts"
          >
            <a
              href="tel:+7987654-54-56"
            >
              +7 987 654-54-56
            </a>
          </div>
          <div
            class="text-contacts"
          >
            <a
              href="tel:+38067504-32-57"
            >
              +38 067 504-32-57
            </a>
          </div>
        </div>
        <div
          class="text-contacts"
        >
          <a
            href="mailto:poss1337@gmail.com"
          >
            Почта: poss1337@gmail.com
          </a>
        </div>
      </div>
      <div
        class="map"
        id="map"
      />
    </div>
  </div>
</DocumentFragment>
`;
